server/models/User.js

```diff
@@ -0,0 +1,72 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema(
+  {
+  // =====================================
+  //  retrieved from google login response
+  // =====================================
+    name: {
+      type: String,
+      minlength: 1,
+      maxlength: 30,
+      trim: true,
+      required: true,
+    },
+    email: {
+      type: String,
+      minlength: 1,
+      maxlength: 30,
+      trim: true,
+      required: true,
+      unique: true
+    },
+    avatarUrl:{
+      type:String,
+      required:true
+    },
+    // =====================================
+    //created by user during onboarding
+    // =====================================
+    timeZone: {
+      type: String,
+      enum: [
+        "UTC-3 ADT",
+        "UTC-4 EDT",
+        "UTC-5 CDT",
```

Comment:
These are the timezones @ylee297 hard coded right? I don't think we need to restrict the strings here, instead just let it be any String cause I think there is hundreds of possibilities in terms of timezones if I'm not incorrect

Reply:
I'm wondering how the calendar would incorporate different time zones later. If we only rely on google calendar, it should be taken care of. But if we want to save the appointments in the database, and check it manually, wouldn't it be too complicated if there were hundreds of them?

Reply:
besides, R we stroting the timezone code ( utc-3, ADT), or the name from moment.js (America/Toronto)?